Dockerfile

```diff
@@ -19,6 +19,7 @@ ENV CONF_D /etc/my.cnf.d
 RUN yum install -y http://www.percona.com/downloads/percona-release/redhat/0.1-3/percona-release-0.1-3.noarch.rpm \
     && yum install -y Percona-XtraDB-Cluster-${INSTALL_PACKAGE_VERSION} \
     && yum install -y iproute socat which nmap-ncat \
```

Comment:
Unfortunately socat is used by the Percona wsrep scripts. I've removed the need for ncat though.
```
[root@pxc-node1-v2-ecmex /]# grep socat /usr/bin/wsrep_sst_xtrabackup-v2
        tfmt='socat'
        wsrep_log_info "Using socat as streamer"
        if [[ ! -x `which socat` ]];then
            wsrep_log_error "socat not found in path: $PATH"
        if [[ $encrypt -eq 2 || $encrypt -eq 3 ]] && ! socat -V | grep -q WITH_OPENSSL;then
            wsrep_log_info "NOTE: socat is not openssl enabled, falling back to plain transfer"
            wsrep_log_info "Using openssl based encryption with socat: with crt and pem"
                tcmd="socat -u openssl-listen:${TSST_PORT},reuseaddr,cert=$tpem,cafile=${tcert}${sockopt} stdio"
                tcmd="socat -u stdio openssl-connect:${REMOTEIP}:${TSST_PORT},cert=$tpem,cafile=${tcert}${sockopt}"
            wsrep_log_info "Using openssl based encryption with socat: with key and crt"
                tcmd="socat -u openssl-listen:${TSST_PORT},reuseaddr,cert=$tpem,key=${tkey},verify=0${sockopt} stdio"
                tcmd="socat -u stdio openssl-connect:${REMOTEIP}:${TSST_PORT},cert=$tpem,key=${tkey},verify=0${sockopt}"
                tcmd="socat -u TCP-LISTEN:${TSST_PORT},reuseaddr${sockopt} stdio"
                tcmd="socat -u stdio TCP:${REMOTEIP}:${TSST_PORT}${sockopt}"
    tfmt=$(parse_cnf sst transferfmt "socat")
        ss -p state listening "( sport = :$PORT )" | grep -qE 'socat|nc' && break
```